src/main/java/io/vertx/core/impl/launcher/commands/BareCommand.java

```diff
@@ -148,8 +173,15 @@ public void run(Runnable action) {
    */
   @SuppressWarnings("ThrowableResultOfMethodCallIgnored")
   protected Vertx startVertx() {
-    MetricsOptions metricsOptions = getMetricsOptions();
-    options = new VertxOptions().setMetricsOptions(metricsOptions);
+    JsonObject optionsJson = getJsonFromFileOrString(vertxOptions, "vertx-options");
```

Comment:
To be updated when the name is finalized.

Reply:
Done